### PR TITLE
Deployment documentation, automation, release fixes, 2.0.0

### LINKED ISSUE #194: Fix deployment scripts
See https://github.com/codingteam/codingteam.org.ru/pull/48#discussion_r84594108.

### LINKED ISSUE #231: evolutions.autocommit is deprecated, use play.evolutions.autocommit instead
From our test log:
```
p.a.Configuration - application.conf @ file:/home/runner/work/loglist/loglist/scalajvm/target/scala-2.13/classes/application.conf: 54: evolutions.autocommit is deprecated, use play.evolutions.autocommit instead
```

### PR DESCRIPTION
**Pull request by ForNeVeR**
Since we've made big incompatible changes in the site (from the deployment side), I've decided to declare this release 2.0.0.

This PR adds deployment automation and refreshes the documentation.

Preview build of 2.0.0 is already deployed to the production server.

**TODO (@ForNeVeR):**
- [x] cleanup test tags pushed to the repository

Closes #231.
Closes #194 (since we have no deployment scripts now).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +87,55 @@ Then, set up the environment variables and run the test suite:
...
+$ docker push codingteam/loglist:latest
+```
+
+Deployment
+----------
+
+To install the application from Docker, run the following command:
+
+```console
+$ docker run -d --restart unless-stopped --name $NAME -v $CONFIG:/app/conf/application.conf --env-file $ENV_FILE --net host codingteam/loglist:$VERSION
```
Comment (Minoru) [inline README.md:116]: `--net host` is pretty powerful, why not just `--publish 9000`?

**Review by Minoru — COMMENTED** [review]
This looks good, but I'm a bit sceptical about the use of `--net host`. Please see my comment below.

File: README.md (hunk 1 above)
Comment (ForNeVeR) [reply README.md:116]: We need `--net host` to acces PostgreSQL instance (which isn't published to the container on our production site).

Any ideas how to solve it without `--net host`? I've read about some magic tricks, but the best of them seem to only work on Docker 20.04 (while we have pretty old version in production).

Comment (Minoru) [reply README.md:116]: It's a bit of a hassle, but we could crate a separate bridge just for this container, and re-configure our PostgreSQL instance to be accessible from that bridge as well. I tried to quickly find an example of this, but apparently people just bind Postgre to bridge0 which is shared by *all* the containers.

Comment (ForNeVeR) [reply README.md:116]: Thanks, I'll take a look.

Comment (ForNeVeR) [reply README.md]: Fixed.

**Comment by ForNeVeR** [comment]
Thanks for review and for help!

**Pull request merged**